.github/workflows/test.yml

```diff
@@ -144,14 +145,53 @@ jobs:
           yarn tailwind-preset build
           yarn components build
       - name: Test Components
-        run: yarn components test-ct --shard ${{ matrix.shard }}
-      - name: Upload Results
-        uses: actions/upload-artifact@v4
+        run: yarn components test-ct --shard=${{ matrix.shardIndex }}/${{ matrix.shardTotal }} --reporter=blob
+      - name: Upload blob report artifacts
         if: failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: blob-report-${{ matrix.shardIndex }}
+          path: packages/orbit-components/blob-report
+          retention-days: 2
+
+  test-ct-report:
+    name: Visual Regression Tests Report
+    runs-on: ubuntu-latest
+    if: always()
+    needs: test-ct
+    env:
+      UPLOAD_REPORT: "false"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install
+        uses: ./.github/actions/node
+      - name: Download blob reports artifacts
+        id: download
+        uses: actions/download-artifact@v4
         with:
-          name: test-results
-          path: packages/orbit-components/test-results/
-          retention-days: 7
+          path: all-blob-reports
+          pattern: blob-report-*
+          merge-multiple: true
+      - name: Merge into HTML Report
+        env:
+          ARTIFACTS_PATH: ${{ steps.download.outputs.download-path }}
+        run: |
+          if [ -z "$(ls -A ${{ env.ARTIFACTS_PATH }})" ]; then
+            echo "No reports found. Exiting."
+            exit 0
+          else
+            npx playwright merge-reports --reporter html ./all-blob-reports
```

Comment:
I know it's documented but it seems a bit obscure that the output folder for this command is `playwright-report/`. Would it make sense to add a comment?

Reply:
I had an echo saying that. But then I noticed that the command itself already echoes it too, so I deleted it.
I will add a comment so it is clear for one that reads the code.